src/components/button/button.test.tsx

```diff
@@ -1,12 +1,51 @@
 import React from 'react';
-import { render, screen } from '@testing-library/react';
+import { render, screen, fireEvent } from '@testing-library/react';
+import '@testing-library/jest-dom';
 
 import Button from './button';
+import { ButtonProps } from './button';
+
+function renderAndGetButton(props: ButtonProps): HTMLElement {
```

Comment:
```suggestion
function sut(props: ButtonProps): HTMLElement {
```

https://github.com/Brisanet/typescript/blob/main/docs/Testes%20Unitarios.md#system-under-test-sut

keep this pattern